packages/flutter/test/material/checkbox_list_tile_test.dart

```diff
@@ -239,4 +239,25 @@ void main() {
 
     expect(tester.widget<InkWell>(find.byType(InkWell)).customBorder, shapeBorder);
   });
+
+  testWidgets('CheckboxListTile.tileColor is passed into the inner ListTile', (WidgetTester tester) async {
```

Comment:
Avoid using language that describes the internal implementation of the widget if possible. For example:
```suggestion
  testWidgets('CheckboxListTile respects tileColor', (WidgetTester tester) async {
```

Reply:
Sir, it may bad to ask a question here, but I will.
- I want to make this kind of contribution but because lack of knowledge, I am not able.
- I just want to know that How can I Learn "How to add tests for any Widget?"